### PR TITLE
Week8

### PR DESCRIPTION
**Pull request by AlenoJankowsky**
PR: #10

### CODE REVIEW

File: week8/homepage/spare-time.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+                if (textAnswer.value === rightFreeAnswer) {
+                    textAnswer.style.backgroundColor = 'green';
+                    document.querySelector('#hobby-text').innerHTML = "Ich habe keine Hobbys (KEKW). Spaß beiseite, meine Hobbys sind: <br> &nbsp Zocken, meine Katze lieb haben, multimediale Unterhaltung.<br/>"
+                    quizDiv.style.display = "block";
+                } else {
+                    textAnswer.style.backgroundColor = 'red';
+                    document.querySelector('#hobby-text').innerHTML = "Einfach faaaaalsch. mmmmmmm."
+                }
+            }
```
Comment (MKrips) [inline week8/homepage/spare-time.html:41]: Please put the script in a separate .js-file

File: week8/trivia/index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
+        <link href="styles.css" rel="stylesheet">
+        <title>Trivia!</title>
+        <script>
+            // TODO: Add code to check answers to questions
```
Comment (MKrips) [inline week8/trivia/index.html:9]: Remove TODO comments if you did that task, otherwise it looks unfinished :)

File: week8/trivia/index.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            document.addEventListener('DOMContentLoaded', function()
+            {
+                let rightDecision = document.querySelector('.rightButton');
+                rightDecision.addEventListener('click', function()
+                {
+                    rightDecision.style.backgroundColor = 'green';
```
Comment (MKrips) [inline week8/trivia/index.html:15]: Please put this in a separate .js-file as well

File: week8/homepage/styles.css [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+  justify-content: left;
+  position: relative;
+  top: 50px;
+}
+
+.resumee-group li {
+  border: 2px solid rgb(77, 57, 57);
+  width:max-content;
+}
+
```
Comment (MKrips) [inline week8/homepage/styles.css:76]: Use BEM for ALL classes in ALL files. I can explain if you need :)

**Review by MKrips — CHANGES_REQUESTED** [review]

File: week8/homepage/resumee.html [hunk 5]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+
+<html lang="de">
+    <head>
```
Comment (DevBastiS) [inline week8/homepage/resumee.html:4]: Indent in HTML files should be 2 spaces.
Please adjust this in your vscode and change the files here.

File: week8/homepage/script.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
+function vanishArrowAndButton() {
+  let arrowElement = document.getElementById("arrow-id");
+  let buttonToShowQuiz = document.getElementById("button-to-show-quiz");
+
+  arrowElement.style.display = "none";
+  buttonToShowQuiz.style.display = "none";
+
```
Comment (DevBastiS) [inline week8/homepage/script.js:7]:
```suggestion
```

**Review by DevBastiS — CHANGES_REQUESTED** [review]

File: week8/homepage/styles.css [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+  font-size: 15pt;
+}
+
+.body__input[type="text"] {
+  line-height: 1;
+  background-color: rgb(108, 88, 88);
+  width: 10%;
+}
+
+.body__eveluate-button {
```
Comment (matthiasdanne) [inline week8/homepage/styles.css:181]:
```suggestion
.body__evaluate-button {
```

**Review by matthiasdanne — APPROVED** [review]
Quite good. 👍

**Review by DevBastiS — APPROVED** [review]

**Pull request merged**